### ISSUE TITLE
Inherit primary header for AstroTable

### ISSUE THREAD

**Issue by pllim** [ejeschke/ginga#405]
Would be nice for `AstroTable` to inherit primary header the same way as `AstroImage`. This is not critical, just nice to have.

**Comment by ejeschke** [comment]
Sounds like it would be straightforward if done in a similar manner to `AstroImage`?

**Comment by pllim** [comment]
Yes, I think with the I/O refactoring done, this will be easier to do now. I just don't have time to look into it yet.
